acceptance/clients/clients.go

```diff
@@ -82,12 +86,15 @@ func AcceptanceTestChoicesFromEnv() (*AcceptanceTestChoices, error) {
 	if externalNetworkID == "" {
 		missing = append(missing, "OS_EXTGW_ID")
 	}
+	if networkID == "" {
+		missing = append(missing, "OS_NETWORK_ID")
+	}
+	if subnetID == "" {
+		missing = append(missing, "OS_SUBNET_ID")
+	}
```

Comment:
This might be a breaking change for OpenLab. The appropriate changes are being made to the `stackenv` script, but I want to make sure that script is being used both in Gophercloud and Terraform testing before merging this PR.

Reply:
How exactly this might be a breaking change for OpenLab?

Reply:
Because this change is now requiring these two environment variables to be set (by adding them to the slice of `missing` variables). So the scripts which execute the acceptance tests need to have these variables set and configured.

Reply:
Ok, i'll just comment out these requirements.

Reply:
You'll also have to make sure the tests for SFS/Manila aren't enabled for the time being. Once it's confirmed that these environment variables have been set in OpenLab, you can have SFS/Manila begin running through OpenLab.

If you want to hold off for another day or two, it shouldn't take long for the OpenLab team to get back to us.

Reply:
They are not enabled, see this comment:

```shell
# Temporarily disabled, see https://github.com/gophercloud/gophercloud/issues/1345
true || go test -v -timeout $timeout -tags "fixtures acceptance" ./acceptance/openstack/sharedfilesystems/v2/
```

Reply:
> If you want to hold off for another day or two, it shouldn't take long for the OpenLab team to get back to us.

We can enable unsupported tests in a new PR.

Reply:
All sounds good - thank you!